worker/uniter/modes.go

```diff
@@ -276,7 +276,7 @@ func ModeAbide(u *Uniter) (next Mode, err error) {
 
 // idleWaitTime is the time after which, if there are no uniter events,
 // the agent state becomes idle.
-var idleWaitTime = 2 * time.Second
+var idleWaitTime = 10 * time.Second
```

Comment:
Why did we change this? 10s is too long IMO. A hook will show executing long after it has finished.